setup.cfg

```diff
@@ -1,8 +1,8 @@
 [metadata]
-name = charmed-openstack-upgrader
+name = canonical-openstack-upgrader
 # See setup.py version generator
-url = https://github.com/esunar/charmed-openstack-upgrader
-description = Application for upgrading Charmed Openstack
+url = https://github.com/esunar/canonical-openstack-upgrader
```

Comment:
why are we pointing to Erhan's fork?

Reply:
Could be a leftover from the previous version of the repo. Corrected it in the new patch